fe/fe-core/src/test/java/com/starrocks/common/TraceManagerTest.java

```diff
@@ -0,0 +1,50 @@
+package com.starrocks.common;
+
+import io.opentelemetry.api.trace.Tracer;
+import org.junit.Before;
+import org.junit.Test;
+
+public class TraceManagerTest {
```

Comment:
I tried setting up WireMock to spin up a mock server with which we could verify if the OpenTelemetry instance is configured correctly to hit tracing endpoints, but I didn't get far with this when I ran into issues with Jetty library resolution. Such an approach would also involve adding dependencies on Jaeger and OpenTelemetry proto files to set up a mock gRPC server, which may be overkill.

Reply:
understand this code is hard to cover indeed, skip it for now.